hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/checknative/CheckNative.java

```diff
@@ -50,12 +56,25 @@ public Void call() throws Exception {
         isalDetail = ErasureCodeNative.getLibraryName();
         isalLoaded = true;
       }
-
     }
     System.out.println("Native library checking:");
     System.out.printf("hadoop:  %b %s%n", nativeHadoopLoaded,
         hadoopLibraryName);
     System.out.printf("ISA-L:   %b %s%n", isalLoaded, isalDetail);
+
+    // Attempt to load the rocks tools lib
+    try {
+      ManagedRawSSTFileReader.loadLibrary();
```

Comment:
Can we use NativeLibraryLoader.load(NativeConstants.ROCKS_TOOLS_NATIVE_LIBRARY_NAME) instead? Since we are checking if the rocks tools library gets loaded.

Reply:
Good idea. Done.